src/patterns/dosdp-patterns/abnormalCell.yaml

```diff
@@ -0,0 +1,33 @@
+pattern_name: abnormalCell
+pattern_iri: http://purl.obolibrary.org/obo/cl/abnormalCell
+
+description: Any abnormal cell type.
```

Comment:
Add more detail. This template is not for any abnormal cell type, e.g. a neoplastic melanocyte. 

```
description: >-
   A grouping class for a variety of abnormal cell types.
```

Reply:
I updated the description to 'A grouping class for a variety of abnormal cell types.'